Package.swift

```diff
@@ -3,12 +3,17 @@ import PackageDescription
 
 var dependencies: [Package.Dependency] = [
     .package(url: "https://github.com/apple/swift-tools-support-core.git", .exact("0.2.3")),
-    .package(url: "https://github.com/apple/swift-argument-parser", "1.0.1"..."1.0.3"),
 ]
 var mockoloFrameworkTargetDependencies: [Target.Dependency] = [
     .product(name: "SwiftSyntax", package: "SwiftSyntax"),
 ]
 
+#if swift(>=5.6)
+dependencies.append(.package(url: "https://github.com/apple/swift-argument-parser", .upToNextMajor(from: "1.0.1")))
```

Comment:
`upToNextMajor(from:)` is deprecated. `from:` should be used.
ref: https://developer.apple.com/documentation/packagedescription/package/dependency/requirement-swift.enum/uptonextmajor(from:)

```suggestion
dependencies.append(.package(url: "https://github.com/apple/swift-argument-parser", from: "1.0.1"))
```

Reply:
Thanks for the review! I've solved the conflicts and updated the method.